components/director/examples/query-tenants/query-tenants.graphql

```diff
@@ -2,6 +2,8 @@
 query {
   result: tenants {
     id
+    internalID
```

Comment:
I just wanted to double-check that it's not a security issue?

Reply:
Well, we do have this field in GraphQL type, so everyone can query it. What I did is including this field in examples, because our approach is to always include all available fields for all examples.